lib/src/credentials.dart

```diff
@@ -28,6 +28,9 @@ enum AuthProviderType {
   /// For authenticating without credentials.
   anonymous,
 
+  /// For authenticating without credentials.
```

Comment:
Should we have more info here. If one looks at the API doc in the IDE it will not be clear how is this different from the other value. Perhaps we could write something like:
```
/// For authenticating without credentials using a new anonymous user.
```